src/assets/assets.js

```diff
@@ -0,0 +1,33 @@
+import doc1 from "./doc1.png.png";
+import doc2 from "./doc2.png.png";
+import doc3 from "./doc3.png.png";
+import doc4 from "./doc4.png.png";
+import doc5 from "./doc5.png.png";
+import doc6 from "./doc6.png.png";
+import doc7 from "./doc7.png.png";
+import doc8 from "./doc8.png.png";
+
+export const specialityData = [
+  { speciality: "All Doctors" },
+  { speciality: "General physician" },
+  { speciality: "Gynecologist" },
+  { speciality: "Dermatologist" },
+  { speciality: "Gastroenterologist" },
+  { speciality: "Neurologist" },
+  { speciality: "Cardiologist" },
+  { speciality: "Orthopaedist" },
+  { speciality: "Physiotherapist" },
+  { speciality: "Internist" },
+  { speciality: "Ophthalmologist" },
+];
+
+export const specialityImages = {
+  "General physician": doc1,
+  Gynecologist: doc2,
+  Dermatologist: doc3,
+  Gastroenterologist: doc4,
+  Neurologist: doc5,
+  Cardiologist: doc6,
+  Orthopaedist: doc7,
+  Physiotherapist: doc8,
+};
```

Comment:
A gdyby tak to wszystko otypować? 

type Speciality = 
  | "All Doctors"
  | "General physician"
  | "Gynecologist"
  | "Dermatologist"
  | "Gastroenterologist"
  | "Neurologist"
  | "Cardiologist"
  | "Orthopaedist"
  | "Physiotherapist"
  | "Internist"
  | "Ophthalmologist";
  
  
interface SpecialityData {
  speciality: Speciality;
}

następnie użyc speciality data do otypowania tablicy?
a na sam koniec mozna specialityImages otypować jako Record<Speciality, string>